src/Compiler.cpp

```diff
@@ -281,7 +281,7 @@ TfLiteNode g_nodes[)"
        << "(TfLiteIntArray*)&" << prefix_ << "tensor_dimension"
        << i << ", ";
     wr << t->bytes << ", ";
-    wr << "\"" << t->name << "\", ";
+    wr << "\"" << ((t->name) ? t->name : "NO-NAME") << "\", ";
```

Comment:
is this perhaps controlled by the macro TF_LITE_STRIP_ERROR_STRINGS ? This would be my way of turning this feature off and on.

Reply:
Nope, it is this one: https://github.com/tensorflow/tensorflow/commit/b6d13bb0a850347f9f885d38b7fbe47c46c494b1#diff-6dace628e5372fb3c3fbb76e0fd9d37a
So the change is fully intentional and this handy debug information will be gone.

Reply:
That sucks!